content/en/getting_started/synthetics/api_test.md

```diff
@@ -122,9 +130,9 @@ The API test and Multistep API test detail pages display an overview of the test
 
 To troubleshoot a failed test, scroll down to **Test Results** and click on a failing test result. Review failed assertions and response details such as status code, response time, and associated headers and body to diagnose the issue.
 
-{{< img src="getting_started/synthetics/api-test-failure.png" alt="API test failure"  style="width:90%;">}}
+{{< img src="getting_started/synthetics/api-test-failure-2.png" alt="API test failure"  style="width:100%;">}}
```

Comment:
Ah nice, this one is indeed way better! Let's just make sure to grab a test run that actually shows 1 as a number of trace (see https://a.cl.ly/Z4uK92Bg). You could for instance clone test 7kz-bt9-ueu from the demo account, slightly tweak the test name and use one of the failing test runs for the screenshot.